lib/compilers/java.js

```diff
@@ -236,7 +236,7 @@ export class JavaCompiler extends BaseCompiler {
             for (const codeLineCandidate of utils.splitLines(codeAndLineNumberTable)) {
                 // Match
                 //       1: invokespecial #1                  // Method java/lang/Object."<init>":()V
```

Comment:
Could we maybe quickly document why the `|default` bit was added to the regex? Maybe something like this
```suggestion
                //       1: invokespecial #1                  // Method java/lang/Object."<init>":()V
                // Or match the "default: <code>" block inside a lookupswitch instruction   
```